src/js/api/justification.js

```diff
@@ -104,3 +105,69 @@ ripe.Ripe.prototype.getJustificationsByContextP = function(context, options) {
         });
     });
 };
+
+/**
+ * Gets the existing justifications, according to the provided filtering
+ * strategy as normalized values.
+ *
+ * Attempts to resolve context, code and full code based on what is given.
+ * Adds those parameters to the filters given.
+ *
+ * If the 'other' code is used, it returns the adhoc text message without
+ * requesting a justification from RIPE Core.

```

Comment:
Perhaps add explanation on why the other logic is different, like (this is just a suggestion). Perhaps also add that explanation in the comment added above the code

```suggestion
 * Gets the existing justifications, according to the provided filtering
 * strategy as normalized values.
 *
 * Attempts to resolve context, code and full code and adds those
 * parameters to the filters provided.
 *
 * If the 'other' keyword is contained in the provided `codeFull`, it means
 * that no default justification was used and the custom text provided is
 * the text that is returned.

```

Reply:
I like it